### PR TITLE
MINOR: Bump timeout for sending records

### PR DESCRIPTION
**Pull request by bbejeck**
Set timeout to be consistent with changes to `2.0` #5841  when merged we should cherry-pick to `2.1`

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by bbejeck** [comment]
@mjsax @vvcephei 

\cc @guozhangwang

**Review by vvcephei — APPROVED** [review]
LGTM. Thanks, @bbejeck !

File: streams/src/test/java/org/apache/kafka/streams/integration/GlobalThreadShutDownOrderTest.java [hunk 1]
```diff
@@ -146,7 +146,7 @@ public void shouldFinishGlobalStoreOperationOnShutDown() throws Exception {
             public boolean conditionMet() {
                 return firstRecordProcessed;
             }
-        }, 10000L, "Has not processed record within 10 seconds");
+        }, "Has not processed record within 15 seconds");
```
Comment (mjsax) [inline streams/src/test/java/org/apache/kafka/streams/integration/GlobalThreadShutDownOrderTest.java]: See my comment on the other PR. Should we increase to 30 seconds?

Comment (bbejeck) [reply streams/src/test/java/org/apache/kafka/streams/integration/GlobalThreadShutDownOrderTest.java]: Yeah, I agree I'll update all related PRs to use a 30 second timeout

**Comment by bbejeck** [comment]
updated this

**Comment by mjsax** [comment]
Java11: https://builds.apache.org/job/kafka-pr-jdk11-scala2.12/302/
```
java.lang.RuntimeException: java.util.concurrent.ExecutionException: org.apache.kafka.common.errors.TimeoutException: Aborted due to timeout.
	at org.apache.kafka.streams.integration.utils.KafkaEmbedded.createTopic(KafkaEmbedded.java:185)
	at org.apache.kafka.streams.integration.utils.EmbeddedKafkaCluster.createTopic(EmbeddedKafkaCluster.java:196)
	at org.apache.kafka.streams.integration.utils.EmbeddedKafkaCluster.createTopics(EmbeddedKafkaCluster.java:160)
	at org.apache.kafka.streams.KafkaStreamsTest.shouldReturnFalseOnCloseWhenThreadsHaventTerminated(KafkaStreamsTest.java:442)
```
This seems to be new? \cc @vvcephei 

4 more non-streams failures:
```
kafka.api.UserClientIdQuotaTest.testThrottledProducerConsumer
kafka.api.UserClientIdQuotaTest.testQuotaOverrideDelete
kafka.api.UserQuotaTest.testThrottledProducerConsumer
kafka.api.UserQuotaTest.testQuotaOverrideDelete
```

Retest this please.

**Comment by bbejeck** [comment]
Non related streams failure `kafka.server.MetadataRequestTest.testAliveBrokersWithNoTopics`

retest this please

**Comment by mjsax** [comment]
Java8 is green. Merging this.

**Comment by bbejeck** [comment]
thanks @mjsax!

**Comment by mjsax** [comment]
Cherry-picked to `2.1` branch.

**Pull request merged**